src/library/Uno.Themes.WinUI.Markup/Theme/CheckBox.cs

```diff
@@ -0,0 +1,254 @@
+					{
+						public static class Foreground
+						{
+							[ResourceKeyDefinition(typeof(Brush), "CheckBoxGlyphForegroundUnchecked")]
+							public static ResourceValue<Brush> Unchecked => new("CheckBoxGlyphForegroundUnchecked", true);
+
+							[ResourceKeyDefinition(typeof(Brush), "CheckBoxGlyphForegroundUncheckedPointerOver")]
+							public static ResourceValue<Brush> UncheckedPointerOver => new("CheckBoxGlyphForegroundUncheckedPointerOver", true);
+
+							[ResourceKeyDefinition(typeof(Brush), "CheckBoxGlyphForegroundUncheckedPressed")]
+							public static ResourceValue<Brush> UncheckedPressed => new("CheckBoxGlyphForegroundUncheckedPressed", true);
+
+							[ResourceKeyDefinition(typeof(Brush), "CheckBoxGlyphForegroundUncheckedDisabled")]
+							public static ResourceValue<Brush> UncheckedDisabled => new("CheckBoxGlyphForegroundUncheckedDisabled", true);
+
+							[ResourceKeyDefinition(typeof(Brush), "CheckBoxGlyphForegroundChecked")]
+							public static ResourceValue<Brush> Checked => new("CheckBoxGlyphForegroundChecked", true);
+
+							[ResourceKeyDefinition(typeof(Brush), "CheckBoxGlyphForegroundCheckedPointerOver")]
+							public static ResourceValue<Brush> CheckedPointerOver => new("CheckBoxGlyphForegroundCheckedPointerOver", true);
+
+							[ResourceKeyDefinition(typeof(Brush), "CheckBoxGlyphForegroundCheckedPressed")]
+							public static ResourceValue<Brush> CheckedPressed => new("CheckBoxGlyphForegroundCheckedPressed", true);
+
+							[ResourceKeyDefinition(typeof(Brush), "CheckBoxGlyphForegroundCheckedDisabled")]
+							public static ResourceValue<Brush> CheckedDisabled => new("CheckBoxGlyphForegroundCheckedDisabled", true);
+
+							[ResourceKeyDefinition(typeof(Brush), "CheckBoxGlyphForegroundIndeterminate")]
+							public static ResourceValue<Brush> Indeterminate => new("CheckBoxGlyphForegroundIndeterminate", true);
+
+							[ResourceKeyDefinition(typeof(Brush), "CheckBoxGlyphForegroundIndeterminatePointerOver")]
+							public static ResourceValue<Brush> IndeterminatePointerOver => new("CheckBoxGlyphForegroundIndeterminatePointerOver", true);
+
+							[ResourceKeyDefinition(typeof(Brush), "CheckBoxGlyphForegroundIndeterminatePressed")]
+							public static ResourceValue<Brush> IndeterminatePressed => new("CheckBoxGlyphForegroundIndeterminatePressed", true);
+
+							[ResourceKeyDefinition(typeof(Brush), "CheckBoxGlyphForegroundIndeterminateDisabled")]
+							public static ResourceValue<Brush> IndeterminateDisabled => new("CheckBoxGlyphForegroundIndeterminateDisabled", true);
+						}
+
+						public static class PathData
+						{
+							[ResourceKeyDefinition(typeof(string), "CheckBoxHyphenGlyphPathData")]
+							public static ResourceValue<string> Hyphen => new("CheckBoxHyphenGlyphPathData", true);
+
+							[ResourceKeyDefinition(typeof(string), "CheckBoxCheckGlyphPathData")]
+							public static ResourceValue<string> Check => new("CheckBoxCheckGlyphPathData", true);
+						}
```

Comment:
We should move this to the Glyph with just properties for `HyphenPathData` and `CheckPathData`

Reply:
removed PathData class and created HyphenPathData and CheckPathData props for Glyph